src/jaxsim/api/model.py

```diff
@@ -1128,10 +1399,12 @@ def step(
     # Step the dynamics forward.
     state_xf, integrator_state_xf = integrator.step(
         x0=state_x0,
-        t0=jnp.array(t0_ns * 1e9).astype(float),
+        t0=jnp.array(t0_ns / 1e9).astype(float),
         dt=dt,
         params=integrator_state_x0,
-        **dict(joint_forces=joint_forces, external_forces=external_forces),
+        **(
+            dict(joint_forces=joint_forces, link_forces=link_forces) | integrator_kwargs
+        ),
```

Comment:
Same suggestion for using dict literal instead of calling `dict`
```suggestion
        **(
            {"joint_forces": joint_forces, "link_forces": link_forces}
            | integrator_kwargs
        ),
```

Reply:
In jit-compiled code, this change has no effect.